app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -98,28 +99,43 @@ fun Scaffold() {
 				}
 			)}
 
-	) { padding ->
```

Comment:
Kotlin coding specs for this do not require a lambda for this, I would recommend removing it.